src/translations/en.json

```diff
@@ -851,7 +851,9 @@
         "cancel": "Cancel",
         "ok": "OK",
         "default_confirmation_title": "Are you sure?",
-        "close": "Close"
+        "close": "Close",
+        "enlarge": "Enlarge",
+        "reduce": "Reduce"
```

Comment:
Similar to my icon comment, "enlarge" is okay I guess but "reduce" doesn't feel right to me.  Again I think the familiar "maximize" and "restore" might be better.